Gordon360/Program.cs

```diff
@@ -49,6 +49,7 @@
 builder.Services.AddScoped<IMembershipRequestService, MembershipRequestService>();
 builder.Services.AddScoped<IAdministratorService, AdministratorService>();
 builder.Services.AddScoped<IEmailService, EmailService>();
+builder.Services.AddScoped<INewsService, NewsService>();
```

Comment:
Why did you add this as a scoped dependency? It's not a wrong thing to do, it just doesn't seem related to your other changes so I'm curious what prompted it.

Reply:
Not sure exactly. It was early on in the process. I can undo it if you would like, but why bother if it's something we're hoping to move towards anyway.